clients/destination.go

```diff
@@ -300,7 +301,20 @@ func (c *DestinationClient) Write2(ctx context.Context, tables schema.Tables, so
 	}
 	_, err = saveClient.CloseAndRecv()
 	if err != nil {
-		return fmt.Errorf("failed to CloseAndRecv client: %w", err)
+		st := status.Convert(err)
+		errorWithDetails := err.Error()
```

Comment:
I'm concatenating errors into one string, but we can decide to do something else